docs/getting_started/why_use_GTFS.md

```diff
@@ -1,42 +1,104 @@
+- **Flexible services:** GTFS can describe demand-responsive options like dial-a-ride, route deviations and, other services that do not follow the common behavior of scheduled and/or fixed service.
+- **Pathways:** GTFS can model large transit stations, helping riders navigate from station entrances and exists to the location where they board or disambark from a transit vehicle.
 
-## GTFS can probably do more than you think
+<div class="grid cards" markdown>
 
-GTFS is mostly known for providing schedule information for public transit, particularly in metro areas with fixed-route services. However, there are optional features beyond the basic GTFS Schedule such as:
+-   :material-train:{ .lg .middle } __What else GTFS can do?__
 
-- [Fares features](/getting_started/features/fares) showing fare costs and structures; 
-- [Flexible services features](/getting_started/features/flexible_services), offering demand-responsive transit options, like dial-a-ride and paratransit services; 
-- [Pathways features](/getting_started/features/pathways), displaying vital accessibility information for rider navigation in stations. 
-- And [many more](/getting_started/features/overview).
 
-Additionally, GTFS data is now being used by a variety of software applications including data visualization and analysis tools for planning and research. Having up-to-date and high quality data provides accurate transit information not just to riders, but also to planners and policymakers who are able to better understand how transit is being used in their communities. 
+    [:octicons-arrow-right-24: Learn more](../features/overview)
 
-## Open Data means more opportunity and choices
+</div>
 
-GTFS is an Open Standard. This means that agencies can make information available using any of the tools which already support GTFS (including simple text editing or a spreadsheet). Open standards lead to the creation of data that can be easily shared. The feeds can be used by trip planners such as Google, Apple, Transit App, Open Trip Planner, and even apps created by riders. Anyone who wants to provide accurate and up-to-date transit information can use GTFS feed to do so.
+## Improved Rider Experience
 
-Some riders like to use different apps depending on their needs—having GTFS lets riders choose what trip planning app suits them best. Some apps may be more accessible or better at providing information for riders with disabilities, some may be simpler and easier to use, and sometimes riders just want the newest app.
+GTFS empowers riders with accurate schedules, real-time updates, and a wider choice of apps, leading to reduced wait times and informed travel decisions. Here's how GTFS enhances the rider experience:
 
-## Maintained by the community
+- **Up-to-Date Information:** GTFS provides riders with access to real-time schedules, routes, and stop information, facilitating informed travel decisions.
```

Comment:
Maybe we can change this line to make it less Realtime-specific (the second point already covers that)? removing "real-time" might be enough to do the trick.

Reply:
I think so too. Plus having real-time schedules makes it sound weird .